### DISCUSSION TITLE
Issue adding additional web pages

### DISCUSSION THREAD

**Discussion by michela** [mozilla/hubs#2985] [General Questions & Discussion] [answered]
Is there any additional setup required to add a new custom page to frontend?

We are trying to add a custom page in the Hubs homepage (a custom Help page). We've created a JS entry file, a HTML file and updated the webpack.config.js file. On the local dev this seems to work great and can we access the page by visiting localhost:8080/newpage but once deployed to our live environment we are getting an invalid URL 404 message when trying to access this new page.

**Comment by robinkwilson** [comment, marked as answer]
The new page needs the new route to be added reticulum (the backend). Since we currently don't have a way to update reticulum to add the new route, we can't render new pages. 

Created an issue in the [github.com/reticulum/hubs-cloud](https://github.com/mozilla/hubs-cloud) repo:
https://github.com/mozilla/hubs-cloud/issues/106
